index.html

```diff
@@ -80,6 +80,231 @@ <h2 class="purple">LET'S CONNECT</h2> <br>
+                                </svg>  
+                            </a>
+                        </li>
+                    </ul>               
+                   </div>
+                   <button class="btn resume-btn" type="button">Get my resume</button>
+                   <ul class="about-list">
+                      <li>
+                        <span> <h3 class="languages-title">Languages</h3>
+                               <img class="vector" src="images/dropdown.png" alt="dropdown">
+                        </span>
+                      </li>
+                       <br> <hr>
+                      <div class="langs">
+                            <ul>
+                                <li>
+                                    <span class="language-container">
+                                        <img src="images/js-logo.png" id="js" alt="JS logo">                                   
+                                        <p>JavaScript</p>
+                                    </span>
+                                </li> <br>
+                                <li>
+                                    <span class="language-container">
```

Comment:
- [ ]  As stated in the project requirement You must stick to the design as much as possible (e.g., font, colors, images, text, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1). but for For each frame work and skill the image are broken as shown below in screenshot  the path you use for image  has no image for js-logo.png, html-logo.png, css-logo.png in your image folder, Kindly Fix it. Please kindly follow the instruction given to you by the previous reviewer i think that will help you  out to fix this broken imags i can see that you have tried alot to fix this but still have some images not display properly.
- [ ] your design 
![you design](https://user-images.githubusercontent.com/92458236/235263311-d4f2fd91-d61e-43d1-b2f8-51ee6d0385ab.jpg)

- [ ] figma design
![design](https://user-images.githubusercontent.com/92458236/235263559-f1b438e3-c91b-4789-a0c1-f54492952f53.jpg)